cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -781,7 +781,7 @@ private boolean getState(HttpServletRequest request, HttpServletResponse respons
       setResponseCode(response, SC_OK);
       out.write(stateString.getBytes(StandardCharsets.UTF_8));
       if (verbose || superVerbose) {
-        out.write(String.format("%n%nMonitored Windows:%n").getBytes(StandardCharsets.UTF_8));
+        out.write(String.format("%n%nMonitored Windows [End-Time (provided), Start-Time(omitted)):%n").getBytes(StandardCharsets.UTF_8));
```

Comment:
Is it more intuitive to just provide the actual format? i.e. Format: [Window End Time=Data_Completeness]